src/Tizen.NUI/src/public/VisualMaps.cs

```diff
@@ -659,24 +659,49 @@ protected virtual void ComposingPropertyMap()
+            PropertyValue mixColor = new PropertyValue(_mixColor);
+            PropertyValue opacity = new PropertyValue((float)_opacity);
+            PropertyValue visualFittingMode = new PropertyValue((int)_visualFittingMode);
+            PropertyValue cornerRa = new PropertyValue((int)cornerRadius);
+            if (_shader != null) { _outputVisualMap.Add(Visual.Property.Shader, shader); }
+            if (_premultipliedAlpha != null) { _outputVisualMap.Add(Visual.Property.PremultipliedAlpha, premultipliedAlpha); }
+            if (_mixColor != null) { _outputVisualMap.Add(Visual.Property.MixColor, mixColor); }
+            if (_opacity != null) { _outputVisualMap.Add(Visual.Property.Opacity, opacity); }
+            if (_visualFittingMode != null) { _outputVisualMap.Add(Visual.Property.VisualFittingMode, visualFittingMode); }
+            if (cornerRadius != null) { _outputVisualMap.Add(Visual.Property.CornerRadius, cornerRa); }
+
+            shader.Dispose();
+            premultipliedAlpha.Dispose();
+            mixColor.Dispose();
+            opacity.Dispose();
+            visualFittingMode.Dispose();
+            cornerRa.Dispose();
         }
 
         private void ComposingTransformMap()
         {
             _visualTransformMap = new PropertyMap();
-            if (_visualSize != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.Size, new PropertyValue(_visualSize)); }
-            if (_visualOffset != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.Offset, new PropertyValue(_visualOffset)); }
-            if (_visualOffsetPolicy != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.OffsetPolicy, new PropertyValue(_visualOffsetPolicy)); }
-            if (_visualSizePolicy != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.SizePolicy, new PropertyValue(_visualSizePolicy)); }
-            if (_visualOrigin != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.Origin, new PropertyValue((int)_visualOrigin)); }
-            if (_visualAnchorPoint != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.AnchorPoint, new PropertyValue((int)_visualAnchorPoint)); }
+            PropertyValue visualSize = new PropertyValue(_visualSize);
+            PropertyValue visualOffset = new PropertyValue(_visualOffset);
+            PropertyValue visualOffsetPolicy = new PropertyValue(_visualOffsetPolicy);
+            PropertyValue visualSizePolicy = new PropertyValue(_visualSizePolicy);
+            PropertyValue visualOrigin = new PropertyValue((int)_visualOrigin);
+            PropertyValue visualAnchorPoint = new PropertyValue((int)_visualAnchorPoint);
+            if (_visualSize != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.Size, visualSize); }
+            if (_visualOffset != null) { _visualTransformMap.Add((int)VisualTransformPropertyType.Offset, visualOffset); }
```

Comment:
could you remove `_` in name of variable? 

in c#, pascal case is supposed to be used and for private member, it would be OK to use `_` prefix. However in NUI, we have decided not to use prefix such as `_`, `m`, `p`, and etc.

- NUI ground rule : https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule
- NUI coding rule : https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=191373542

could you use PascalCase even for private fields?